.circleci/config.yml

```diff
@@ -181,19 +181,16 @@ commands:
             pip install -r test-requirements.txt
       - run:
           name: download manager docker image
-          command:  wget http://repository.cloudifysource.org/cloudify/5.1.0/.dev1-release/cloudify-manager-aio-docker-5.1.0.dev1.tar
-      - run:
-          name: load docker image
-          command: docker load -i cloudify-manager-aio-docker-5.1.0.dev1.tar
-      - run:
-          name: retain space by dumping the tar
-          command: rm cloudify-manager-aio-docker-5.1.0.dev1.tar
+          command:  docker pull cloudifyplatform/premium-cloudify-manager-aio:latest
```

Comment:
Its tricky. I like the way it was before because when we release 5.2 for example so this image will be 5.2 image and the name of the test is 5.1.

Reply:
We are primarily concerned about testing py2 vs py3 wagons. the 510 thing is secondary to that. we're not going to start testing every version of Cloudify - it is just too resource intensive.

Reply:
OK